### ISSUE TITLE
Verbesserung Feldkauf

### ISSUE THREAD

**Issue by HurrycaneBE** [cloudmaker97/FS25-Discord-Bot#16]
hey , wäre es möglich anzeigen zu lassen wenn jemand ein feld kauft bzw verkauft ? Auch eine Übersicht in dieser Hinsicht wäre nice für einen Admin um die Einhaltung von Regeln zu überprüfen ;)

**Comment by cloudmaker97** [comment]
Ja, das sollte möglich sein. Leider befinde ich mich mitten im Umzug, daher habe ich gerade kaum Kapazitäten das selbst zu implementieren. Aber jeder darf hier seine Veränderungen mit einbringen :)
